linux_os/guide/services/base/service_kdump_disabled/rule.yml

```diff
@@ -52,6 +52,10 @@ ocil_clause: |-
 ocil: |-
     {{{ ocil_service_disabled(service="kdump") }}}
 
+fixtext: '{{{ fixtext_service_disabled("kdump") }}}'
+
```

Comment:
There is an extra newline appearing at the beginning of the OCIL text in the generated guide.